test/Sema/implementation-only-import-inlinable-conformances.swift

```diff
@@ -0,0 +1,263 @@
+// RUN: %empty-directory(%t)
```

Comment:
Can you add some tests where a base class has an implementation-only conformance to a protocol, and the derived class appears in a substitution map somewhere against a conformance requirement to the protocol? This is where you get an `InheritedProtocolConformance` and we should make sure that works too.

Reply:
I think I had them in the other file (the non-inlinable tests) but I'll add some here too.